HACKING.md

```diff
@@ -4,70 +4,84 @@ Hacking on `snapd` is fun and straightforward. The code is extensively unit
 tested and we use the [spread](https://github.com/snapcore/spread)
 integration test framework for the integration/system level tests.
 
-## Development
+## Setting up
 
 ### Supported Ubuntu distributions
 
 Ubuntu 18.04 LTS or later is recommended for `snapd` development.
+Usually, the latest LTS would be the best choice.
 
-If you want to build or test on older versions of Ubuntu, additional steps
+>If you want to build or test on older versions of Ubuntu, additional steps
 may be required when installing dependencies.
 
 
 ### Supported Go version
 
-Go 1.13 or later is required to build `snapd`.
+[Go 1.13](https://pkg.go.dev/go@go1.13) or later is required to build `snapd`.
 
-If you need to build older versions of snapd, please have a look at the file
-`debian/control` to find out what dependencies were needed at the time
+>If you need to build older versions of snapd, please have a look at the file
+[debian/control](debian/control) to find out what dependencies were needed at the time
 (including which version of the Go compiler).
 
-### Setting up your build environment
+### Setting up build environment
```

Comment:
These two lines were brought by diff by accident, since somewhere here in the previous version was a section with the similar heading, apparently.  

Regarding _your_ or _the_, I personally prefer _the_. First of all, _your_ form doesn't appear in any of the headings in this document. Second, it feels to me that the heading reflects a purpose not to declare a request. I mean when we give guidelines, it is acceptable to provide guidelines in a form like: "do this", "take your", etc. From my experience (as far as I understand it) this form is less preferable for headings.  

@mardy  and @MiguelPires, would it be okay with you if I added _the_, not _your_?

Reply:
I'm fine with either :-)